### PR TITLE
Add the ability to use client certificates.

### PR DESCRIPTION
**Pull request by qbit**
This is often called CertFP by various services.

### CODE REVIEW

File: irken.tcl [hunk 1]
```diff
@@ -652,10 +652,11 @@ namespace eval ::irken {
             return
         }
         set insecure [dict get? 0 $::config $serverid -insecure]
+        set certfile [dict get? "" $::config $serverid -certfile]
         set port [dict get? [expr {$insecure ? 6667:6697}] $::config $serverid -port]
 
         addchantext $serverid "Connecting to $serverid ($host:$port)..." -tags system
-        set chan [if {$insecure} {socket -async $host $port} {tls::socket -async $host $port}]
+        set chan [if {$insecure} {socket -async $host $port} {if {$certfile ne ""} {tls::socket -certfile $certfile -async $host $port } {tls::socket -async $host $port }}]
```
[the remark this answers is not in the document]
Comment (qbit) [reply irken.tcl]: Updated!

**Comment by dlowe-net** [comment]
Thanks for the addition!

**Comment by qbit** [comment]
NP, thanks for making a fun project :D!

**Comment by ilmari-lauhakangas** [comment]
@dlowe-net I noticed this and some other commits are only in the master branch and not in the main branch, I suppose by accident?

**Pull request merged**